Generator/Source/CuckooGeneratorFramework/Utils.swift

```
@@ -18,6 +18,17 @@ extension String {
         return components(separatedBy: occurence).first
     }
 
+    func matches(for regex: String) -> [String] {
+      do {
```

You may omit the `do catch`, because as far as I know, `NSRegularExpression.init` throws only when the regex is invalid, in which case I think it's better to throw as well instead of silently returning "no matches" array.